apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -94,6 +98,21 @@ const VideoPreviews: React.FC<{
   );
 };
 
+const getNativePropsMessage = (elements: SBEnumType['value']): JSX.Element => {
```

Comment:
Let's put a bit more craft to this:
- Move it under props table
- Use nicer component 

<img width="1008" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/4593cd2b-107a-426d-860b-4d5c53a0ce66">


---

```
  nativeProps: {
    display: 'flex',
    gap: tokens.spacingHorizontalM,

    border: `1px solid ${tokens.colorNeutralStroke1}`,
    borderRadius: tokens.borderRadiusMedium,
    padding: tokens.spacingHorizontalM,
    margin: `0 ${tokens.spacingHorizontalM}`,
  },
  nativePropsIcon: {
    alignSelf: 'center',
    color: tokens.colorBrandForeground1,
    fontSize: '24px',
  },
  nativePropsMessage: {
    display: 'flex',
    flexDirection: 'column',
    gap: tokens.spacingVerticalXS,
  },
```

```
const getNativeElementsList = (elements: SBEnumType['value']): JSX.Element => {
  const elementsArr = elements.map((el, idx) => [
    <code key={idx}>{`<${el}>`}</code>,
    idx !== elements.length - 1 ? ', ' : ' ',
  ]);

  return (
    <>
      {elementsArr}
      {elementsArr.length > 1 ? 'elements' : 'element'}
    </>
  );
};
```

```
          {primaryStory.argTypes.as && primaryStory.argTypes.as?.type?.name === 'enum' && (
            <div className={styles.nativeProps}>
              <InfoFilled className={styles.nativePropsIcon} />
              <div className={styles.nativePropsMessage}>
                <b>
                  Native props are supported <span role="presentation">🙌</span>
                </b>
                <span>
                  All HTML attributes native to the {getNativeElementsList(primaryStory.argTypes.as.type.value)},
                  including all <code>aria-*</code> and <code>data-*</code> attributes, can be applied as native props
                  on this component.
                </span>
              </div>
            </div>
          )}
```

Reply:
@layershifter thanks! definitely looks better 👍🏻